src/frontend/src/lib/stores/token-group.store.ts

```diff
@@ -0,0 +1,7 @@
+import { initCertifiedSetterStore } from '$lib/stores/certified-setter.store';
+
+export interface TokenGroupData {
+	expanded: boolean;
+}
+
+export const tokenGroupStore = initCertifiedSetterStore<TokenGroupData>();
```

Comment:
@AntonioVentilii-DFINITY but those kind of data are never certified? or is there plan to save the information in the backend?